enhancements/single-node/single-node-openshift-with-workers.md

```diff
@@ -0,0 +1,583 @@
+---
+title: single-node-openshift-with-workers
+authors:
+  - "@omertuc"
+reviewers:
+  - "@romfreiman"
+  - "@eranco74"
+  - "@tsorya"
+  - "@dhellmann"
+  - "@Miciah"
+  - "@bparees"
+  - "@JoelSpeed"
+  - "@staebler"
+  - "@derekwaynecarr"
+approvers:
+  - TBD
```

Comment:
The enhancement is open for almost 3 months already. We're about to miss the deadline. If you do see some critical points, please raise them asap. Otherwise, I really would like to merge it till the end of the week